clusterloader2/testing/l4ilb/dep.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.Name}}
+spec:
+  replicas: {{RandIntRange .ReplicasMin .ReplicasMax}}
```

Comment:
Do we want to keep RandIntRange (random number from range [ReplicasMin; ReplizacMax])? If not, we can use a single variable.